src/pygeoprocessing/geoprocessing_core.pyx

```diff
@@ -878,8 +880,9 @@ def _raster_band_percentile_double(
         pixels_processed += block_data.size
         if time.time() - last_update > 5.0:
             LOGGER.debug(
-                'data sort to heap %.2f%% complete',
-                (100.*pixels_processed)/n_pixels)
+                f'data sort to heap {(100.*pixels_processed)/n_pixels}% '
```

Comment:
Same with formatting the percentage float:
... can we format this float percentage doing something like `f'{value:.2f}'`. So:
`f'data sort to heap {((100.*pixels_processed)/n_pixels):.2f}% '`

Reply:
Sure, I put .1 precision since I've always found 2 digits of percent precision annoying except i used to do it all the time. fixed in 4dab73968a828f6e247de289468e32e361b99d90